src/cmds/qstat.c

```diff
@@ -3407,3 +3435,21 @@ cnvt_est_start_time(char *est_time, int wide)
 
 	return timebuf;
 }
+
+/**
+ * @brief used to display server instance failures in case of  MULTI_SERVER
+ * 
+ * @return void
+ */
+static void
+show_svr_inst_fail(int fd)
```

Comment:
Don't we need this function in several commands? If so, we could move this to libcmds and reused from all commands